lms/djangoapps/certificates/tests/test_signals.py

```diff
@@ -261,7 +278,11 @@ def test_cert_generation_on_photo_verification_instructor_paced(self):
                     status='submitted'
                 )
                 attempt.approve()
-                mock_generate_certificate_apply_async.assert_called_with(kwargs={
-                    'student': unicode(self.user_two.id),
-                    'course_key': unicode(self.course_two.id),
-                })
+                mock_generate_certificate_apply_async.assert_called_with(
+                    countdown=CERTIFICATE_DELAY_SECONDS,
+                    kwargs={
+                        'student': unicode(self.user_two.id),
+                        'course_key': unicode(self.course_two.id),
+                        'expected_verification_status': SoftwareSecurePhotoVerification.STATUS.approved
+                    }
+                )
```

Comment:
I highly suggest refactoring the tests in this file as part of the refactoring effort.  There is quite a bit of repetition that can be eliminated with `ddt`.  For example, the only difference between this test and the one above it is using `user_one`/`course_one` versus `user_two`/`course_two`.  As a reader, it takes time to find that difference.  With `ddt` it will reduce duplication and make it easier to read.
Similar refactoring can be done on the tests above.